static/js/search.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { Component } from "react";
```

Comment:
Since there are multiple searchboxes on the site, could you rename this to timeline_search.tsx?

Bonus: move all the timeline + scatter related js/ts/tsx files to a tools subdirectory. Can do that in a followup.

Reply:
Renamed the file, I will move the files in the next PR.